data-prepper-plugins/kafka-source/README.md

```diff
@@ -0,0 +1,150 @@
+# Kafka source
+
+This is the Data Prepper Kafka source plugin that reads records from Kafka topic. It uses the consumer API provided by Kafka to read messages from the broker.
+
+The Kafka source plugin supports OpenSearch 2.0.0 and greater.
+
+## Usages
+
+The Kafka source should be configured as part of Data Prepper pipeline yaml file.
+
+## Configuration Options
+
+```
+log-pipeline:
+  source:
+    kafka:
+      bootstrap_servers:
+        - 127.0.0.1:9092,localhost:9093, 127.0.0.1:9094
+      topics:
+        - topic:
```

Comment:
I dont know if we have agreed upon having per topic configs, but I see lot of config is common and repetitive for multiple topics. Auth config probably shouldn't  be topic specific.

Reply:
We should make `topics` a list of strings.

```
topics:
  - my-topic-2
  - my-topic-3
```

This also equivalent to:

```
topic: [ "my-topic-2", "my-topic-3" ]
```

In code, this will be represented by:

```
private List<String> topic;
```